webpack/assets/javascripts/react_app/routes/Models/ModelsPage/components/ModelDeleteModal.js

```diff
@@ -16,11 +15,8 @@ const ModelDeleteModal = props => {
       enforceFocus
       submitProps={{
         url: `/api/v2/models/${id}`,
```

Comment:
I think `id` is coming in `undefined` here.  When I click submit on the delete confirmation dialog, 
the DELETE request is going to `/api/v2/models/undefined`

and I'm getting an error:
```
Resource model not found by id 'undefined'
```

Reply:
Interesting, I did not run into that. The correct `id` is always set when I click the delete button in the table. I remove